### DISCUSSION TITLE
Could Backpex have a UI component/Form project as a standalone dependency?

### DISCUSSION THREAD

**Discussion by lucasrabaca** [naymspace/backpex#656] [Ideas]
I'm new to Phoenix and am not aware of any other project that offers all these features—such as forms with relations, real-time search, etc in components that can be used for general purposes outside of the admin.

Or even import some of backpex modules to replicate the form/fields generation outside of the admin logic?

**Comment by Flo0807** [comment]
Hey there!

We do not plan to move our components into a standalone dependency at the moment. But our goal is to make all Backpex components as generic as possible in the future.

Nevertheless, you could possibly install Backpex together with daisyUI and Alpine.js and use only the Backpex UI components. You do not need to generate admin views.

So you might want to look at the `Backpex.HTML.*` modules ([here](https://github.com/naymspace/backpex/tree/main/lib/backpex/html)) which expose most of our UI components.

Note that these components don't provide any interactivity, such as real-time search.

If you are looking for a full-featured UI component library, there are other projects that may be more suitable.
